src/components/ProfileHeaderLogo/ProfileHeaderLogo.test.tsx

```diff
@@ -0,0 +1,14 @@
+import { cleanup, render, screen } from "@testing-library/react";
+import { afterEach } from "vitest";
+
+import { ProfileHeaderLogo } from "./ProfileHeaderLogo";
+
+afterEach(() => {
```

Comment:
The global test setup file already has this `afterEach` cleanup block.  This one can be removed.